docs/mp/guides/overview.adoc

```diff
@@ -97,7 +97,48 @@ in your Helidon MP application.
 --
 Learn how to build a Helidon MicroProfile (MP) application from scratch.
 --
-====
+
+[CARD]
+.Helidon MP Upgrade guide
+[link=migration.adoc]
+--
+Learn how Upgrade your Helidon MP application.
+--
+
+[CARD]
+.OIDC Tutorial
+[link=security-oidc.adoc]
+--
+Learn how to set up OIDC security in your Helidon MP application.
+--
+
+[CARD]
+.Helidon MP Tracing
+[link=tracing.adoc]
+--
+Learn how to use tracing in your Helidon MP application.
+--
+
+[CARD]
+.Testing with JUnit 5
+[link=testing-junit5.adoc]
+--
+Helidon has great integration with JUnit5, learn how to use it for testing your Helidon application.
```

Comment:
Learn how to use JUnit5 for teating your applications.

Reply:
Fixed.